Modified Network view graph

LedaLima opened the pull request:
Altered the parameters for the Force graph to spread the nodes a little bit more on its panel

Commit:
Altered the parameters for the Force graph to spread the nodes a little bit more on its panel

rabarona commented on ui/dns/js/components/NetworkPanel.react.js:
```diff
@@ -145,8 +145,8 @@ function filterDataAndBuildGraph()
-                          .charge(-Math.round(h * 0.45)) // 0.45 is a magic number for graph styling purposes charge is 45% of the grap height
-                          .linkDistance(Math.round(h * 0.081)) // 0.081 is a magic number for graph styling purposes linkDistance is 8.1% of the graph height
+                          .charge(-Math.round(w * 0.45)) // 0.45 is a magic number for graph styling purposes charge is 45% of the grap height
+                          .linkDistance(Math.round(w * 0.081)) // 0.081 is a magic number for graph styling purposes linkDistance is 8.1% of the graph height
```
is w for width? In that case I'd update the comment next to the code line as well.

EverLoSa commented:
LGTM

daortizh commented:
LGTM :+1:

LedaLima committed:
Update NetworkPanel.react.js

Updated the inline comment accordingly to the change performed, now the layout is using the width of the area to set charge and link Distance

rabarona commented:
L G T M
